docs/cicd.md

```diff
@@ -2,14 +2,60 @@
 
 ## Content
 
+- [Deliver a new release by triggering the CI](#triggering-a-delivery-using-the-ci)
 - [Workflows](#workflows)
+- [Services](#services)
 - [Fastlane](#fastlane)
 
+## Triggering a Delivery using the CI
+
+The easiest way to deliver a new build to production or development is to trigger the corresponding CircleCI workflows _triggered_native_development_delivery_ and _triggered_production_delivery_:
+
+- Get a CircleCI [Personal API Token](https://circleci.com/docs/2.0/managing-api-tokens/).
+- Trigger a delivery by sending a POST request to:`https://circleci.com/api/v2/project/github/digitalfabrik/lunes-app/pipeline`
+  - header needs to contain `Circle-Token=your-personal-token` and `content-type=application/json`
+  - body could look like this:
+    ```json
+    {
+      "parameters": {
+        "run_commit": false,
+        "run_dev_delivery": true
```

Comment:
Shouldn't we rather just use one (enum) parameter? Otherwise I think this could get quite tedious to specify all other params.